### PR TITLE
Fix Pypi HomePage Link

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/setup.py [hunk 1]
```diff
@@ -69,7 +69,7 @@
     license='MIT License',
     author='Microsoft Corporation',
     author_email='ascl@microsoft.com',
-    url='https://github.com/Azure/azure-storage-python',
+    url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob',
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-blob/setup.py]: Not sure if we want to keep use https://github.com/Azure/azure-sdk-for-python or the detailed one. Any thoughts?

Comment (rakshith91) [reply sdk/storage/azure-storage-blob/setup.py]: We might want to use the main link..that's what other packages are doing.

Comment (xiafu-msft) [reply sdk/storage/azure-storage-blob/setup.py]: I just took a look at azure-identity package and they are using the detailed one https://pypi.org/project/azure-identity/
so probably let's use the detailed one for all?

Comment (rakshith91) [reply sdk/storage/azure-storage-blob/setup.py]: hmm, sounds good..in that case, lets change for queues and files as well :)

Comment (xiafu-msft) [reply sdk/storage/azure-storage-blob/setup.py]: we have already did I think?

**Pull request merged**